docs/v1/user-story.md

```diff
@@ -0,0 +1,124 @@
+# A Year In The Life Of A Kubernetes Service Developer
+
+## 0. Today
+I'ma developer in an enterprise environment.  For some fun in my spare time, I’m building a node.js chat application for me 
+and a couple of coworkers to goof around with.  It’s really simple.  It runs on one container.  To get going, I simply 
+download a node.js container onto minikube on my laptop and started coding.  I want some co-workers to be able to find the 
```

Comment:
Isn't that how the story goes (more or less)?

1. Narrator makes the story in local kube
    - no types
    - no brokers
    - no service catalog (unless I created confusion with him using the existing "service" construct)

2. Narrator uses RabbitMQ
    - Though I did have the app on the corporate kube already which makes sense - chat app should not die when narrator closes laptop

3. Narrator registers in service catalog
    - That only happens at the point where someone wants to bind to it (tech support team)


Am I misunderstanding?